package.json

```
@@ -1,28 +1,47 @@
 {
   "name": "imgix-core-js",
-  "description": "Common boilerplate for all imgix JavaScript-based functionality.",
+  "description": "Core imgix URL and srcset builder library.",
```

Super nitpick but I don't want to forget to mention it. Let's change the package description to match the rest of the SDK.
```suggestion
  "description": "A JavaScript client library for generating image URLs with imgix",
```